src/styles/signup.scss

```diff
@@ -0,0 +1,135 @@
+.signup {
+    margin: 0;
+    display: flex;
+    min-height: 100vh;
+    justify-content: center;
+    align-items: center;
+    background: #E5E5E5;
+    font-family: "Poppins", sans-serif,  'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;  
```

Comment:
There seems to be a repetition in the font family here and in your `Login.scss`, is it possible to add a global style where your font-family definition will exist?